### PR TITLE
Code clean-up

### PR DESCRIPTION
**Pull request by NElnour**
I:

- re-formatted code

- fixed typo in _draw_snake_body

- added self. designation for all Field class attributes

- removed dead instantiations

### CODE REVIEW

File: field.py [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+from snake import Snake
 
 
 class Field:
     """A field class represents the state of the field
     It also updates and displays the field
     The size of a field is 300 x 300 and each grid should have size 10 x 10
     === Attributes ===
-    _snake: an instance of snake in the field
+    -snake: an instance of snake in the field
```
Comment (shogi9x9) [inline field.py]: private attribute has to start with "_" but not "-" so could you please fix it?

File: field.py [hunk 2]
```diff
@@ -20,7 +21,7 @@ class Field:
     _appleX: int
     _appleY: int
 
-    def __init__(self,game_surface):
+    def __init__(self, game_surface):
```
Comment (shogi9x9) [inline field.py]: you might want to add type annotations? 
it will be pygame.Surface

File: field.py [hunk 3]
```diff
@@ -33,7 +34,7 @@ def display_field(self) -> None:
         Clear the field first and create a new field
         TODO: write a doctest
         """
-        window.fill((255,255,255)) #clear field
+        self.window.fill((255, 255, 255))  # clear field
```
Comment (shogi9x9) [inline field.py]: Do you think it is better keep color to be constant? (you can define constant by such as COLOR_WHITE)

File: game.py [hunk 4]
```diff
@@ -21,6 +21,9 @@ class Game:
     _menu_index: int
     _field: Optional[Field]
 
+    def __init__(self, l: int, w: int) -> None:
+        """Initializes a snake class and field class"""
+
```
Comment (shogi9x9) [inline game.py]: Init function seems to be duplicated here. I changed the window size and therefore arguments seems to be no longer needed.

File: snake.py [hunk 5]
```diff
@@ -9,7 +10,7 @@ class Snake:
     When he hits the wall or eats his tail, the game is over.
 
     === Attributes ===
-    _head: represents snake's head. this must be the first element in _body.
+    _head: represents snake's head. this must be the first element in _body
     _body: represents snake's body.
```
Comment (shogi9x9) [inline snake.py]: you might want to remove period on this line as well?

Comment (NElnour) [reply snake.py]: Hey, wouldn't periods in docstrings be fine? It's just that we can't add them to commit messages

**Pull request closed without merging**